lib/v2/tisi/radar.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
         '.': [
             {
                 title: '最近更新',
-                docs: 'https://docs.rsshub.app/new-media.html#topys-guan-jian-zi',
+                docs: 'https://docs.rsshub.app/new-media.html#teng-xun-yan-jiu-yuan-zui-jin-geng-xin',
```

Comment:
This should point to the H2 header, not the H3 header

Reply:
I think H3 link is clearer than H2 link.

Reply:
https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-docs

Reply:
OK